starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <title>Apple Pie Recipe</title>
+    <link rel="stylesheet" href="./css/main.css" />
+    <link href="https://fonts.googleapis.com/css?family=Public+Sans&display=swap" rel="stylesheet">
+  <body>
+    <div class="header-container">
+    <h1>Apple Pies</h1>
+  </div>
+  <div class="content">
+    <section class="introduction">
+    This was my grandmother's apple pie recipe. I have never seen another one
+    quite like it. It will always be my favorite and has won me several first
+    place prizes in local competitions. I hope it becomes one of your favorites
+    as well! <br>
+    <img src="/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/recipe-info.png" alt="recipe info">
```

Comment:
I don't see the images when I clone your project so make sure to write the src as I showed you before :)

the correct path is:
`src="./images/recipe-info.png"`